.github/workflows/ci.yml

```diff
@@ -115,8 +115,8 @@ jobs:
     strategy:
       fail-fast: true
       matrix:
-        python-version: ["3.7", "3.8", "3.9"]
-        nautobot-version: ["1.2.11", "1.3.7"]
```

Comment:
There is dependency problem with Nautobot 1.2.11 and the mkdocs-material (8.4.2).
I assumed it won't be a big problem to move to the highest supported version of Nautobot (1.3.x). The Nautobot versions 1.4.x and 1.5.x are failing. There is a bug for 1.5 that should cover both.

Reply:
If you still want to support Nautobot 1.2.x you could downgrade the `mkdocs-material` version to a compatible one (it's a Jinja2 2.x vs 3.x conflict that's the cause).

Reply:
ok, let's try it

Reply:
I downgraded `mkdocs-material` to 8.3.0 so it can work with nautobot 1.2.11 - I've added that back to the matrix, so it's back to where we started.

I've removed python 3.10 from the matrix since it fails with 1.2.11 and it's beyond the scope of this PR to fix that :)